airgun/views/job_invocation.py

```diff
@@ -28,125 +27,89 @@ def is_displayed(self):
 
 class JobInvocationCreateView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
-    job_category = FilteredDropdown(id='job_invocation_job_category')
-    job_template = FilteredDropdown(locator="//div[contains(@class, 'job_template_selector')]")
-    bookmark = FilteredDropdown(id='targeting_bookmark')
-    search_query = TextInput(name='targeting[search_query]')
-    template_content = ConditionalSwitchableView(reference='job_template')
-
-    @template_content.register('Run Command - SSH Default', default=True)
-    class RunSSHCommandForm(View):
-        command = TextInput(id='command')
-
-    @template_content.register('Power Action - SSH Default')
-    class RestartHostForm(View):
-        action = FilteredDropdown(id='s2id_action')
 
-    @template_content.register('Puppet Run Once - SSH Default')
-    class RunPuppetForm(View):
-        puppet_options = TextInput(id='puppet_options')
+    class category_and_template(WizardStepView):
```

Comment:
Can we please use the python naming convention here for class names, using either camelcase class names and no underscores in names? same for other classes
```suggestion
    class CategoryAndTemplate(WizardStepView):
```

Reply:
Also, for consistency with other non-class view items, it may be better to keep it as it is. Because in robottelo, you just view it as a "values" dict and it would be awful to have some keys lowercase and some uppercase. Hard decision.